pandas/tests/tseries/offsets/test_business_hour.py

```diff
@@ -231,8 +231,9 @@ def test_add_datetime(
         assert offset8 + dt == datetime(2014, 7, 1, 11)
         assert offset9 + dt == datetime(2014, 7, 1, 22)
         assert offset10 + dt == datetime(2014, 7, 1, 1)
+        assert dt + 0 * offset1 == dt
```

Comment:
Could you make this into a separate test instead?

Reply:
Sure, I've made a separate test for the zero multplications.